test/e2e/far_e2e_test.go

```diff
@@ -116,30 +117,41 @@ var _ = Describe("FAR E2e", func() {
 			nodeBootTimeBefore, err = e2eUtils.GetBootTime(clientSet, nodeName, testNsName, log)
 			Expect(err).ToNot(HaveOccurred(), "failed to get boot time of the node")
 
-			// create tested pod, and save its creation time
-			// it will be deleted by FAR CR
+			// create tested pod which will be deleted by the far CR
 			pod = e2eUtils.GetPod(nodeName, testContainerName)
 			pod.Name = testPodName
 			pod.Namespace = testNsName
 			Expect(k8sClient.Create(context.Background(), pod)).To(Succeed())
-			log.Info("Tested pod has been created", "pod", testPodName)
-			creationTimePod = metav1.Now().Time
 			DeferCleanup(cleanupTestedResources, pod)
 
 			// set the node as "unhealthy" by disabling kubelet
 			makeNodeUnready(selectedNode)
 
+			startTime = time.Now()
 			far := createFAR(nodeName, fenceAgent, testShareParam, testNodeParam)
-			DeferCleanup(deleteFAR, far)
+			DeferCleanup(deleteFAR, startTime, far)
 		})
 		When("running FAR to reboot two nodes", func() {
 			It("should successfully remediate the first node", func() {
-				checkRemediation(nodeName, nodeBootTimeBefore, creationTimePod, pod)
+				checkRemediation(nodeName, nodeBootTimeBefore, pod)
+				remediationTimes = append(remediationTimes, time.Since(startTime))
 			})
 			It("should successfully remediate the second node", func() {
-				checkRemediation(nodeName, nodeBootTimeBefore, creationTimePod, pod)
+				checkRemediation(nodeName, nodeBootTimeBefore, pod)
+				remediationTimes = append(remediationTimes, time.Since(startTime))
 			})
 		})
+		AfterEach(func() {
+			if len(remediationTimes) == 2 {
+				averageTimeDuration := 0.0
+				for index, remTime := range remediationTimes {
+					averageTimeDuration += remTime.Seconds()
+					fmt.Printf("\nRemediation time #%d: %s\n", index+1, remTime)
```

Comment:
please use the logger `log` instead of `fmt.Print`

Reply:
I see that have used fmt.Print in more places in the e2e test file, so I will update them all.